tests/konsist/src/test/kotlin/io/element/android/tests/konsist/KonsistFlowTest.kt

```diff
@@ -24,7 +24,12 @@ class KonsistFlowTest {
         val regex = "(.*)\\)(\n\\s*)*\\.collectAsState".toRegex()
 
         val allowedMethods = listOf(
-            "isOnline()",
+            "syncService.isOnline()",
+            "userListDataStore.selectedUsers()",
+            "accountProviderDataSource.flow()",
+            "syncService.isOnline()",
+            "timeline.paginationStatus",
+            "analyticsService.getUserConsent()",
```

Comment:
Maybe using an annotation to suppress the warning would be better, but I don't know if that's possible/easy. Just something we could keep in mind for future usages of Konsist.